src/installation/thirdparty.md

```diff
@@ -8,6 +8,42 @@ Those methods are not officially supported, if you encounter installation issues
 
 A [Powershell script](https://github.com/losslesspng/SetUpKomgaJava) to get up and running with Komga.
 
+## Scoop (Windows)
+
+Komga is available in [Scoop](https://github.com/ScoopInstaller/Scoop)'s [extras](https://github.com/ScoopInstaller/Extras) bucket.
+
+### Installation
+
+#### 1. (Skip if Scoop is installed) Install Scoop
```

Comment:
I think it's better to mention you need Scoop, and provide a link to the scoop's website, which will always have up to date installation instructions, rather than adding that step here

Reply:
Ah yea, make sense.